src/get_model_metadata_impl.cpp

```diff
@@ -21,18 +21,22 @@ using google::protobuf::util::JsonPrintOptions;
 using google::protobuf::util::MessageToJsonString;
 
 namespace ovms {
-
 Status GetModelMetadataImpl::getModelStatus(
```

Comment:
Instead of creating method overload, we might take ModelManager as parameter with default set to `ModelManager::getInstance()` if not specified.